src/App.jsx

```
@@ -1,58 +1,62 @@
-import UserAuth from "./User/pages/UserAuth/UserAuth";
-import AboutUs from "./User/pages/AboutUs/Aboutus.jsx";
-import Contributors from "./User/pages/Contributors/Contributors.jsx";
-import Contact from "./User/pages/Contacts/Contact";
-import Dashboard from "./User/pages/Dashboard/Dashboard";
-import Home from "./User/pages/Home/Home";
+// Layouts & shared
 import UserLayout from "./User/UserLayout";
 import Team from "./User/components/Team/Team";
-import Confirmation from "./User/pages/Confirmation/Confirmation";
-import Cancellation from "./User/pages/Cancellation/Cancellation";
+import Error from "./User/pages/404-Page/Error";
```

Biome (2.3.13)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)